src/migrations/tasks/20240718222006_swu-evaluation-tables.ts

```diff
@@ -92,6 +92,46 @@ export async function up(connection: Knex): Promise<void> {
     '
   );
 
+  const opportunitiesWithStatusAndStatusId = await connection(
+    "swuOpportunities as opportunities"
+  )
+    .join(
+      connection.raw(
+        "(??) as statuses",
+        connection("swuOpportunityStatuses")
+          .select("opportunity", "id", "status")
+          .rowNumber("rn", function () {
+            this.orderBy("createdAt", "desc").partitionBy("opportunity");
+          })
+      ),
+      function () {
+        this.on("opportunities.id", "=", "statuses.opportunity").andOnNotNull(
+          "statuses.status"
+        );
+      }
+    )
+    .where({
+      "statuses.rn": 1
+    })
+    .select("opportunities.id", "statuses.id as statusId", "statuses.status");
+
+  const inProgressStatuses = opportunitiesWithStatusAndStatusId.reduce(
+    (inProgressStatuses, { statusId, status }) => {
+      return status === PreviousSWUOpportunityStatus.EvaluationTeamQuestions
+        ? [...inProgressStatuses, statusId]
+        : inProgressStatuses;
+    },
+    []
+  );
+  await connection("swuOpportunityStatuses")
+    .update({ status: SWUOpportunityStatus.EvaluationTeamQuestionsIndividual })
+    .where("id", "in", inProgressStatuses);
+
+  // Maintain deprecated status for historical purposes
+  await connection("swuOpportunityStatuses")
+    .update({ status: SWUOpportunityStatus.DeprecatedEvaluationTeamQuestions })
```

Comment:
@dhruvio I think we might need to change all statuses to SWUOpportunityStatus.EvaluationTeamQuestionsIndividual as well as adding a SWUOpportunityStatus.EvaluationTeamQuestionsConsensus, because this [check](https://github.com/bcgov/digital_marketplace/pull/504/files#diff-3e3de5abd29971dad9a6471f8a13824bda480401b562ac5014ed6d3f63fb237eL102) requires the opportunity to have the status in its history to view them.

Reply:
Took a stab at addressing this in https://github.com/bcgov/digital_marketplace/pull/504/commits/1dc501c56feaa5e9a789818037294c43f7d14314; we can revert if necessary.